src/main/java/org/czekalski/petapiintegration/apiClient/v1/dto/AddressDto.java

```diff
@@ -0,0 +1,77 @@
+
+package org.czekalski.petapiintegration.apiClient.v1.dto;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.*;
+
+
+public class AddressDto {
+
+
+
+    private String address1;
+
+    private final String address2;
+
+    private final String city;
+
+    private final String state;
+
+    private final String postcode;
+
+    private final String country;
+
+    private Map<String, Object> additionalProperties = new HashMap<String, Object>();
+
+    @JsonAnyGetter
+    public Map<String, Object> getAdditionalProperties() {
+        return this.additionalProperties;
+    }
+
+    @JsonAnySetter
+    public void setAdditionalProperty(String name, Object value) {
+        this.additionalProperties.put(name, value);
+    }
+
+    @JsonCreator
+    public AddressDto(  @JsonProperty("address1")String address1,
```

Comment:
Ja zazwyczaj umieszczam w takich przypadkach parametry już w nowej linii, bo tak nazwa klasy spycha je na prawo i ciężej się to czyta. Tak zawsze są w tym samym miejscu.

Reply:
Ta uwaga dotyczy wszystkich dto